src/forms/forms.service.ts

```
@@ -180,7 +180,10 @@ export class FormsService {
     let query = this.formRepository
       .createQueryBuilder('form')
       .select(['form.formid', 'form.title', 'form.status', 'form.fields'])
-      .where('form.context = :context', { context: whereClause.context });
+      .where('form.context = :context', { context: whereClause.context })
```

ESLint

[error] Replace `'form.context·=·:context'` with `"form.context·=·:context"`

(prettier/prettier)